abci/example/kvstore/kvstore.go

```diff
@@ -437,10 +436,15 @@ func (app *Application) execPrepareTx(tx []byte) *types.ExecTxResult {
 // proposal for transactions with the prefix strips.
```

Comment:
This was pre-existing, but we're in here anyway.

```suggestion
// proposal for transactions with the prefix stripped.
```